README.md

```diff
@@ -3,6 +3,7 @@
 [ZeroMQ](http://zeromq.org/) - Low-level networking library for building connected software.
 
 [![Platforms](http://img.shields.io/badge/platforms-ubuntu-lightgrey.svg?style=flat)](#)
+[![Platforms](https://travis-ci.org/telusdigital/ansible-zeromq.svg?branch=master)](#)
```

Comment:
`[![Build Status](https://travis-ci.org/telusdigital/ansible-zeromq.svg?branch=master)](https://travis-ci.org/telusdigital/ansible-zeromq)`

Build Status not Platform